src/pages/api/trpc/[trpc].ts

```diff
@@ -15,14 +14,10 @@ export default trpcNext.createNextApiHandler({
   /**
    * @link https://trpc.io/docs/error-handling
    */
-  onError({ error, path }) {
-    // This is logger re-creation is needed because trpc does not know if the logger middleware has been reached up till this point
-    // path can also be undefined if any error was thrown before that
-    const logger = createBaseLogger(path ?? 'no path found')
-
+  onError({ error }) {
     if (error.code === 'INTERNAL_SERVER_ERROR') {
       // send to bug reporting
-      logger.error('Something went wrong', error)
+      console.error('Something went wrong', error)
```

Comment:
@karrui i undid this change and added an error log statement at the logger middleware instead. I thought that was better for code traceability purposes. With that we could actually just remove this line, lmk what you think